src/components/RadioButton/RadioButton.types.ts

```diff
@@ -0,0 +1,11 @@
+import { ReactNode } from 'react';
+
+import { DefaultComponentProps } from '../../types';
+
+export interface RadioButtonProps extends DefaultComponentProps {
+	checked?: boolean;
+	disabled?: boolean;
+	id?: string;
+	label: string | ReactNode;
+	onChange?: () => void;
```

Comment:
Mss hier de event nog meegeven als parameter, als dat voor de Checkbox ook niet is mag je dat daar ook aanpassen.